app/controllers/app.rb

```diff
@@ -21,36 +25,58 @@ class App < Roda
         view 'home'
       end
 
+      # GET /404
       routing.on '404' do
-        routing.get do
-          view '404'
-        end
+        view '404'
       end
 
+      # GET /resource_nil
       routing.on 'resource_nil' do
         view 'resource_nil'
       end
 
       # Inbox
       routing.on 'inbox' do
-        routing.on 'new' do
+        
+        # GET /inbox/new-inbox
+        routing.on 'new-inbox' do
           view 'new_inbox'
         end
 
+        # GET /inbox/guest-inbox
+        routing.on 'guest-inbox' do
+          
+          # Get guest suggestions
+          session[:guest_suggestions] ||= []
+          
+          session[:guest_suggestions].insert(0, "suggestion1").uniq!
+          session[:guest_suggestions].insert(0, "suggestion2").uniq!
+          # session[:guest_suggestions].delete("suggestion1")
+          # session[:guest_suggestions].delete("suggestion2")
+          puts "guest_suggestions = #{session[:guest_suggestions]}"
+
+          view 'guest_inbox', locals: { guest_suggestions: session[:guest_suggestions] }
```

Comment:
Can we call it 'suggestions' rather than 'guest_suggestions'?

Reply:
Ok, done.